casper/contracts/simple_casper.v.py

```diff
@@ -103,16 +104,9 @@
 # Base penalty factor
 base_penalty_factor: public(decimal)
 
-# Current penalty factor
-current_penalty_factor: public(decimal)
-
```

Comment:
@jonchoi I think that you wanted to change this to `reward_factor` but didn't? I don't see where `reward_factor` is initialized

Reply:
Ooopsy I'm blind: https://github.com/jonchoi/casper/blob/e891459a174ec7f6e8687f960980306900cdd6c9/casper/contracts/simple_casper.v.py#L99